web/client/utils/MapInfoUtils.js

```diff
@@ -91,6 +159,37 @@ export const getLayerFeatureInfoViewer = (layer) => {
 export const getLayerFeatureInfo = (layer) => {
     return layer && layer.featureInfo && {...layer.featureInfo} || {};
 };
+/**
+ * @return {string} the info format value from layer, otherwise the info format in settings
+ */
+
```

Comment:
For this method, props object is not necessary, only format is necessary. Anyway it is ok also this way. Just document it poperly.
```suggestion
/**
 * Extracts the proper mime time to use for the layer, given the passed props that determine the preferred type. This 
 *  helps to convert, for instance, the mime-type set as default for the map (e.g. `application/json`) into the effective
 * mime type requested by the server (e.g. `application/geo+json`)
 * @param {object} layer the layer
 * @param props.format the preferred format. it can be a mime type like `application/json`
 * @return {string} the info format value from layer, otherwise the info format in settings
 */
```